REORG_LOSSES.md

```diff
@@ -0,0 +1,19 @@
+# Reorg losses
+
+From time to time ethereum network forks for a short time causing block reorgs.
```

Comment:
Please grammar/typo check.

Also might add a link to https://etherscan.io/blocks_forked